task/11-katas-2-tasks.js

```diff
@@ -34,7 +34,21 @@
  *
  */
 function parseBankAccount(bankAccount) {
-    throw new Error('Not implemented');
+  let result = '';
+  let numbers = {
+    top: [[' _ '], ['   '], [' _ '], [' _ '], ['   '], [' _ '], [' _ '], [' _ '], [' _ '], [' _ ']],
+    middle: [['| |'], ['  |'], [' _|'], [' _|'], ['|_|'], ['|_ '], ['|_ '], ['  |'], ['|_|'], ['|_|']],
+    bottom: [['|_|'], ['  |'], ['|_ '], [' _|'], ['  |'], [' _|'], ['|_|'], ['  |'], ['|_|'], [' _|']]
+  }
+  for (let i = 0; i < 27; i += 3) {
+    let top = bankAccount.slice(i, i + 3);
+    let middle = bankAccount.slice(i + 28, i + 31);
+    let bottom = bankAccount.slice(i + 56, i + 59);
+    for (let j = 0; j < 10; j++) {
+      if (top == numbers.top[j] && middle == numbers.middle[j] && bottom == numbers.bottom[j]) result += j;
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
'0' == 0
and '0' === 0)"